incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/model/OpenApi.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.model;
+
+import java.util.List;
+import java.util.Map;
+
+public class OpenApi
+{
+    public String openapi;
+    public List<Server> servers;
+    public Map<String, PathItem> paths;
+    public Components components;
```

Comment:
Please make sure all the classes have `Openapi` prefix in their class names.

Reply:
you mean like `OpenapiComponents`,  `OpenapiPathItem`